### PR TITLE
Layout framework selection

### PR DESCRIPTION
**Pull request by Jefiozie**
This PR will introduce a better visual way to select one of the generators.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xlayers/vscode-extension",
+  "name": "xlayers-extension",
```
Comment (manekinekko) [inline package.json]: What's the reason for this change?

Comment (Jefiozie) [reply package.json]: We can't use scoped names here. We can use any name as long it is not scoped. Do you have a suggestion for the name?
